lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserSwiftDescriptorFinder.cpp

```diff
@@ -0,0 +1,225 @@
+//
+//===-- DWARFASTParserSwiftDescriptorFinder.cpp -----------------*- C++ -*-===//
```

Comment:
replace ------*- C++ -*- with ----------------
This only makes sense for `.h` files, to tell editors that it is C++ and not C